src/visualization/types/Map/view.tsx

```diff
@@ -28,7 +28,7 @@ const GeoPlot: FC<Props> = ({result, properties}) => {
 
   const {lat, lon} = properties.center
 
-  if (!getLatLon(result.table, 0)) {
+  if (!getGeoCoordinates(result.table, 0)) {
```

Comment:
like i said in the previous review, this will crash and not work the way you expect. it doesn't show the error message you have below, it shows the generic error boundary page and throws an error in the console.

it's pretty easy to test, and i expect you to as part of your normal PR process. any non map data will cause this error to trigger. 

here's how i fixed it locally:

```js
try {
  getGeoCoordinates(result.table, 0)
} catch {
  const error =
    'Map type is not supported with the data provided: Missing latitude/longitude values'

  return (
    <div className="panel-resizer--error" data-testid="geoplot-error">
      {error}
    </div>
  )
}
```